client/src/partials/subsidies/subsidies.html

```diff
@@ -21,32 +21,32 @@
 
       <div class="col-md-6">
         <div class="panel panel-default" style="overflow: auto; max-height: 500px;">
-          <div class="panel-heading">
-            {{ "SUBSIDY.ALL_SUBSIDIES" | translate }}
+          <div class="panel-heading" translate>
+            SUBSIDY.ALL_SUBSIDIES
           </div>
 
           <table class="table table-condensed">
             <thead>
               <tr>
-                <th>{{ "TABLE.COLUMNS.NR" | translate}}</th>
-                <th>{{ "TABLE.COLUMNS.LABEL" | translate}}</th>
-                <th>{{ "TABLE.COLUMNS.VALUE" | translate}}</th>
-                <th>{{ "TABLE.COLUMNS.ACCOUNT" | translate}}</th>
-                <th colspan="2">{{ "TABLE.COLUMNS.ACTION" | translate }}</th>
+                <th translate> TABLE.COLUMNS.NR </th>
+                <th translate> TABLE.COLUMNS.LABEL </th>
+                <th translate> TABLE.COLUMNS.VALUE </th>
+                <th translate> TABLE.COLUMNS.ACCOUNT </th>
+                <th colspan="2" translate> TABLE.COLUMNS.ACTION </th>
               </tr>
             </thead>
             <tbody>
               <tr ng-repeat="subsidy in SubsidyCtrl.subsidies | orderBy:'text'" ng-class="{'rowSelected' : SubsidyCtrl.subsidy.id === subsidy.id}">
                 <td>{{ $index + 1 }}</td>
                 <td>{{ subsidy.label }}</td>
-                <td>{{ subsidy.value }}</td>
+                <td>{{ subsidy.value }} %</td>
```

Comment:
The extra space here looks a bit weird.  Why don't you change this to `{{ subsidy.value }}%`.